test/test_utils/imageutils.py

```diff
@@ -0,0 +1,100 @@
+import re
+import boto3
+import json
+
+def get_image_account_id(image_uri):
+    """
+    Find the account ID where the image is located
+    :param image_uri: <str> ECR image URI
+    :return: <str> AWS Account ID
+    """
+    return image_uri.split(".")[0]
+
+def get_image_repository_name(image_uri):
+    repository_uri, _ = image_uri.split(":")
+    _, repository = repository_uri.split("/")
+    return repository
+
+def get_image_tag_name(image_uri):
+    _, tag = image_uri.split(":")
+    return tag
```

Comment:
We already have an equivalent function in test_utils/init.py.

https://github.com/aws/deep-learning-containers/blob/5f02453540ccf56862355b6864a746a2765fd4ed/test/test_utils/__init__.py#L1485-L1494

Is this meant to replace that function? Should it the other function be removed and replaced everywhere else?

Reply:
Same here as above.